app/handlebars-helpers.js

```diff
@@ -4,29 +4,22 @@
  */
 var commonmark = require('commonmark');
 var config = require('./config');
+var getObject = require('./get-object');
 var urljoin = require('url-join');
 
 var reader = new commonmark.Parser();
 var writer = new commonmark.HtmlRenderer();
 
+function getViewProps(metadata, activeView) {
+    var db = getObject(metadata, 'extensions.jupyter_dashboards');
+    return db && db.views[activeView];
+}
+
 module.exports = {
     config: function(name) {
         return config.get(name);
     },
 
-    defaultDashboardConfig: function(layout) {
-        if (layout === 'grid') {
-            return {
-                layout: 'grid',
-                cellMargin: config.get('DB_CELL_MARGIN'),
-                defaultCellHeight: config.get('DB_DEFAULT_CELL_HEIGHT'),
-                maxColumns: config.get('DB_MAX_COLUMNS')
-            };
-        } else {
-            return {};
-        }
-    },
```

Comment:
@dalogsdon: Do we need to have defaults for these values? Or will all created dashboards have it? What about notebooks with no layout?

If we don't need these any more, then we can remove from **config.json**, right?